### ISSUE TITLE
Issues with Google Scholar: OSF claims to be a journal?

### ISSUE THREAD

**Issue by wetneb** [CenterForOpenScience/osf.io#7739]
From a dissemin user:

> I recently deposited the author proof versions of my published papers into the repositories Zendoo, Hal, and OSF. But it looks like that OSF changes the paper citation in a way like that it published in a journal named OSF. I even received some updates in my google scholar account that it suggests changing the name of journals to OSF. THIS IS A HUGE MESS. This does not happen with other repositories.

**Comment by wetneb** [comment]
This is the email they have received:
![suggested_edits](https://user-images.githubusercontent.com/309908/30916077-602b742e-a390-11e7-8390-547da0995e24.png)

This really does not send out the right message! Dissemin provided OSF with the Springer DOI for this paper, so OSF has all the information it needs to generate the correct citation.

**Comment by nicipfeiffer** [comment]
@wetneb Is this the OSF preprint you are referring to, https://osf.io/srjpy/?
I am working on understanding the issue.

**Comment by wetneb** [comment]
@nicipfeiffer yes exactly! Thank you for looking into that!

**Comment by wetneb** [comment]
@nicipfeiffer good catch! I was wrong to claim that we had provided the DOI for all four projects. I will check with the user to know if they also got notifications for other papers, or just this one.

However, notice that while the citation does contain the publisher DOI when provided, it still displays "Open Science Framework" as the publisher, while I would expect to read the publisher's name instead (granted, OSF is much more of a "publisher" than the paywall-manufacturing-industry, but users still expect to see the journal names in citations).

This is true for both the citation displayed in the UI and the citation exposed via the meta tags:
`<meta class="ember-view" name="citation_publisher" content="Open Science Framework" id="ember1074">`

That's why I expect that Google Scholar might still propose the publisher update even when the publisher DOI is provided. But I hope to be wrong.

**Comment by wetneb** [comment]
Quoting the user:

> I have received such notification just for this paper. But the citations in OSF website are wrong for all papers and I suppose the Google robot will catch them sooner or later.

So I am probably wrong. Google Scholar seems to be clever enough in this case! However, the user agrees that "citations in OSF website are wrong", so our point about that still holds.

**Comment by wetneb** [comment]
@nicipfeiffer The user also mentions that the paper does not actually have any DOI, so we could not have provided any.

**Comment by BRosenblatt** [comment]
@wetneb, we're in the process of updating postprint citations to be more reflective of the original publication. I'll add your feedback to our internal tracker and will let you know when citation improvements go out. If you do not have a DOI to share, you could add the journal publication in the abstract as a temporary workaround.

**Comment by wetneb** [comment]
@BRosenblatt that's awesome, thank you so much!

**Comment by BRosenblatt** [comment]
@wetneb I wanted to reach out to let you know that you can now include the original publication date for you work! Simply edit your preprint, and open the "Basics" section. Then enter the date in the "Original publication date" field. The date will be included the citations.

Enjoy!

**Comment by wetneb** [comment]
@BRosenblatt fantastic! thanks again!
